### DISCUSSION TITLE
eQSL exchange does not work

### DISCUSSION THREAD

**Discussion by Jan324663** [magicbug/Cloudlog#1031] [Q&A] [answered]
Hello,

I am using the latest version of Cloudlog. Unfortunately, I am not able to exchange data with eQSL.
I get the following error messages:

Your eQSL username and/or password is incorrect.
OR:
No such Username/Password found This could mean the wrong callsign or the wrong password, or the user does not exist.

I have entered my QTH nickname and my eQSL password here:

![grafik](https://user-images.githubusercontent.com/84246983/118375218-a2867080-b5c0-11eb-8901-0b6ba1cae6a3.png)

What can I do?
Thank you!

**Comment by dg9vh** [comment, marked as answer]
Just Enter your Login-Callsign and put the QTH-Handle into the
Station-Profile or where it is actually located in Cloudlog.
73 de Kim
DG9VH

Am Sa., 15. Mai 2021 um 21:02 Uhr schrieb Jan324663 <
***@***.***>:

> Hello,
>
> I am using the latest version of Cloudlog. Unfortunately, I am not able to
> exchange data with eQSL.
> I get the following error messages:
>
> Your eQSL username and/or password is incorrect.
> OR:
> No such Username/Password found This could mean the wrong callsign or the
> wrong password, or the user does not exist.
>
> I have entered my QTH nickname and my eQSL password here:
>
> [image: grafik]
> <https://user-images.githubusercontent.com/84246983/118375218-a2867080-b5c0-11eb-8901-0b6ba1cae6a3.png>
>
> What can I do?
> Thank you!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1031>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OXBP6ENZB56DI5FHO3TN3ANHANCNFSM446E7KWA>
> .
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen

**Reply by Jan324663** [reply]
Hello Kim,

thanks for the hint. Now it works!

Jan (DO8DX)

**Reply by magicbug** [reply]
@dg9vh now Station Location but you was in the right area :)

**Comment by magicbug** [comment]
Kim was spot on and pleased it fixed it
